python/agents/product-catalog-ad-generation/content_gen_agent/utils/gemini_utils.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for interacting with the Gemini API."""
+import logging
+from typing import Any, Dict, List, Optional
+
+from google import auth
+from google import genai
+from google.api_core import exceptions as api_exceptions
+from google.genai import types
+from google.genai.types import HarmBlockThreshold, HarmCategory
+
+from content_gen_agent.utils.evaluate_media import (
+    evaluate_media,
+)
+
+IMAGE_MIME_TYPE = "image/png"
+
+SAFETY_SETTINGS = [
+    types.SafetySetting(
+        category=HarmCategory.HARM_CATEGORY_HATE_SPEECH,
+        threshold=HarmBlockThreshold.OFF,
+    ),
+    types.SafetySetting(
+        category=HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT,
+        threshold=HarmBlockThreshold.OFF,
+    ),
+    types.SafetySetting(
+        category=HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT,
+        threshold=HarmBlockThreshold.OFF,
+    ),
+    types.SafetySetting(
+        category=HarmCategory.HARM_CATEGORY_HARASSMENT,
+        threshold=HarmBlockThreshold.OFF,
+    ),
+]
+
+GENERATE_CONTENT_CONFIG = types.GenerateContentConfig(
+    temperature=1.0,
+    top_p=0.95,
+    safety_settings=SAFETY_SETTINGS,
+    image_config=types.ImageConfig(
+        aspect_ratio="9:16",
+    )
+)
+
+async def call_gemini_image_api(
+    client: genai.Client,
+    model: str,
+    contents: List[Any],
+    image_prompt: str,
+) -> Dict[str, Any]:
```

Comment:
Looks like this function returns a dictionary. Can you update the return type here (as well as other similar cases)?